ebml_matroska.xml

```diff
@@ -398,7 +398,7 @@ If this Element is used, then any Language Elements used in the same TrackEntry
 see [@!I-D.ietf-cellar-codec] for more info.</documentation>
   </element>
   <element name="CodecPrivate" path="\Segment\Tracks\TrackEntry\CodecPrivate" id="0x63A2" type="binary" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Private data only known to the codec.</documentation>
+    <documentation lang="en" purpose="definition">Private data only known to the codec. This element **MUST NOT** be present if the codec mapping specification defines no initialization or an initialization `none`, else **MUST** be present.</documentation>
```

Comment:
IMO present and with a size of 0 would also mean 'none'. For example in VP9 you can have one to give the 10/12 bits profiles. But there may be none. IMO writing an empty field is better than no field at all in this case. It's clear that it's the default profile. (there are tons of 10 bits VP9 files with no CodecPrivate and there's no way to tell if they are 8 or 10 bits).

This is also in line with
> If the encoding does require any form of Initialization then `CodecPrivate Element` **MUST** be written

`V_PRORES` would probably benefit from that as well.

Reply:
poke

IMO this addition is not good, it's up to the codec to tell what it means if it's present or not.